android/engine/src/main/java/org/smartregister/fhircore/engine/datastore/mockdata/SerializablePractitionerDetails.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021-2023 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.engine.datastore.mockdata
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class SerializablePractitionerDetails(
```

Comment:
This should be migrated to the `shared.models` directory in the next phase of code refactoring. Could you rename them accordingly?

Reply:
done